proxy/http/HttpSM.cc

```diff
@@ -6890,7 +6890,8 @@ HttpSM::update_stats()
     int offset = 0;
     int skip = 0;
 
-    t_state.hdr_info.client_request.url_print(url_string, sizeof url_string, &offset, &skip);
+    t_state.hdr_info.client_request.url_print(url_string, sizeof(url_string), &offset, &skip);
+    url_string[sizeof(url_string) - 1] = 0; // NULL terminate the string
```

Comment:
The int HTTPHdr::url_print(char *buff, int length, int *offset, int *skip) only called here, can we define the output buffer ( buff ) is a '\0' terminated string?